src/photosui.cs

```diff
@@ -474,7 +478,6 @@ interface PHProjectTypeDescriptionInvalidator
 	[iOS (8,0)]
 	[NoMac][NoTV]
 	[DisableDefaultCtor]
-	[Unavailable (PlatformName.MacCatalyst)]
```

Comment:
This doesn't look correct, the `PHEditingExtensionContext` type doesn't show up for Mac Catalyst: https://developer.apple.com/documentation/photokit/pheditingextensioncontext?language=objc

If it turns out it actually is available (the documentation is wrong), then you need to remove the advice on the next line too.

Reply:
I'm sorry, you are right! I had added the MacCatalyst there and since removed it, but forgot to add that Unavailable back in!